### DISCUSSION TITLE
Virustotal scan

### DISCUSSION THREAD

**Discussion by Meshuggahlad** [audiamus/BookLibConnect#78] [Q&A] [answered]
Hi Audiamus, thank you for the great software! I was a bit concerned when I just put BookLibConnect-0.10-Setup.exe into virustotal.com and 2 sites marked out as malicious/trojan. Do you know why that is?
Thank you

**Comment by audiamus** [comment, marked as answer]
No idea. The way antivirus software works is to look for patterns in the binaries and in rare cases genuine software can trigger such an alert. But there may me more detailed info from your virus checker why a particular package is marked by the tool.
